aemcmc/gibbs.py

```diff
@@ -82,31 +178,89 @@ def horseshoe_step(
     ..[1] Makalic, Enes & Schmidt, Daniel. (2016). High-Dimensional Bayesian
           Regularised Regression with the BayesReg Package.
     """
-    upsilon_inv = srng.exponential(1 + lambda2_inv)
-    zeta_inv = srng.exponential(1 + tau2_inv)
+    upsilon_inv = srng.exponential(1 + lmbda_inv)
+    zeta_inv = srng.exponential(1 + tau_inv)
 
     beta2 = beta * beta
-    lambda2_inv_new = srng.exponential(upsilon_inv + 0.5 * beta2 * tau2_inv / sigma2)
-    tau2_inv_new = srng.gamma(
+    lmbda_inv_new = srng.exponential(upsilon_inv + 0.5 * beta2 * tau_inv / sigma)
+    tau_inv_new = srng.gamma(
         0.5 * (beta.shape[0] + 1),
-        zeta_inv + 0.5 * (beta2 * lambda2_inv_new).sum() / sigma2,
+        zeta_inv + 0.5 * (beta2 * lmbda_inv_new).sum() / sigma,
     )
-    return lambda2_inv_new, tau2_inv_new
+    return lmbda_inv_new, tau_inv_new
 
 
-def horseshoe_nbinom(
-    srng: at.random.RandomStream,
-    X: TensorVariable,
-    y: TensorVariable,
-    h: TensorVariable,
-    beta_init: TensorVariable,
-    local_shrinkage_init: TensorVariable,
-    global_shrinkage_init: TensorVariable,
-    n_samples: TensorVariable,
-) -> Tuple[List[Variable], Dict]:
-    r"""
-    Build a symbolic graph that describes the gibbs sampler of the negative
-    binomial regression with a HorseShoe prior on the regression coefficients.
+# sigmoid ------------------------------------------------------------
```

Comment:
```suggestion
# sigmoid ------------------------------------------------------------
```

We need to avoid comment based structural indicators; they introduce various style choices that we don't want to make unless they're necessary.  Also, in many cases, their use can indicate that something is fundamentally wrong with the package/module layout or the naming of objects, and those issues should be addressed directly instead.

Reply:
I agree. The package layout will probably change soon, but it should be fine for now even without these comments.